### PR TITLE
Генералов Александр - Лабораторная работа #1 

### PR DESCRIPTION
**Pull request by AleksandrGeneralov**

### CODE REVIEW

**Comment by AleksandrGeneralov** [comment]
@SemenDoc
@kkonsw

File: code/vector_3d/vector_3d.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+class Vector3d:
+
+    def __init__(self, x: float = 1.0, y: float = 1.0, z: float = 1.0):
+        self.x = x
+        self.y = y
+        self.z = z
+
+    def is_equal(self, x: float, y: float, z: float) -> bool:
+        return self.x == x and self.y == y and self.z == z
```
Comment (kkonsw) [inline code/vector_3d/vector_3d.py:13]: Use small value to compare floats:
```python
epsilon = 0.000001
return abs(a - b) <= epsilon
```
Python 3.5 adds the `math.isclose()` and `cmath.isclose()` functions which tell whether two values are approximately equal or “close” to each other.

**Review by kkonsw — CHANGES_REQUESTED** [review]

**Comment by kkonsw** [comment]
:+1:

**Comment by SemenDoc** [comment]
👍

**Pull request merged**